src/providers/recordset.tsx

```diff
@@ -300,6 +302,7 @@ export default function RecordsetProvider({
   const [isLoading, setIsLoading, isLoadingRef] = useStateRef<boolean>(!initialPage);
   const [pageLimit, setPageLimit, pageLimitRef] = useStateRef(typeof initialPageLimit === 'number' ? initialPageLimit : RECORDSET_DEFAULT_PAGE_SIZE);
   const [page, setPage, pageRef] = useStateRef<any>(initialPage ? initialPage : null);
+  const [pagingSuccess, setPagingSuccess] = useState<boolean>(false);
```

Comment:
Is there a reason why this is not implemented using custom events? Similar to what we've done for "row edit", "row delete", etc? You can see how `src/components/record.tsx` is already acting upon the existing custom events. We should just add another custom event for clicking on next/prev.

I think using events is much better than adding this state variable. The variable is set to true when users click on next/previous and then is set to false when we're doing the scroll. So this is more or less is implemented as an event handler already just without the actual event.